extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/transaction/MongoTransactionException.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.mongodb.panache.transaction;
+
+public class MongoTransactionException extends RuntimeException {
```

Comment:
This doesn't appear to be used.

Reply:
This is used to encapsulate the JTA SystemException into a RuntimeException here: https://github.com/quarkusio/quarkus/pull/14002/files#diff-d003b221f9b819da503954b1f03c63b8ddc7fb18b78c6921cc9ae4d9e1483d1aR370